ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/util/JsonMapper.java

```diff
@@ -1,46 +0,0 @@
-/*
- * Copyright 2020 Confluent Inc.
- *
- * Licensed under the Confluent Community License (the "License"); you may not use
- * this file except in compliance with the License.  You may obtain a copy of the
- * License at
- *
- * http://www.confluent.io/confluent-community-license
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations under the License.
- */
-
-package io.confluent.ksql.api.client.util;
-
-import com.fasterxml.jackson.core.JsonGenerator;
-import com.fasterxml.jackson.databind.DeserializationFeature;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.node.JsonNodeFactory;
-import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
-
-public final class JsonMapper {
```

Comment:
Did this get replaced?

I'm asking since there are a bunch of little overrides below.

Reply:
Nope, it wasn't being used anywhere AFAIK (see `ApiJsonMapper`, which I believe is the one that's used more consistently)